emscripten.py

```diff
@@ -2662,6 +2664,7 @@ def create_receiving_wasm(exports, initializers):
         # With assertions off, hot-swap implementation to avoid garbage via
         # arguments keyword.
         receiving.append('''\
+/** @type {function(...*):?} */
```

Comment:
The above three annotations basically say that things we export from asm.js/wasm module are functions that take in a variadic number of arguments, and we don't know what the functions return.

To improve this check "tighter" in the future, we could look at generating per-function specific annotations with proper argument counts and return value types.